### ISSUE TITLE
Display Preferences not saving

### ISSUE THREAD

**Issue by jezzom** [jellyfin/jellyfin-androidtv#2598]
### Describe the bug

Hi!

I've been using and enjoying Jellyfin for about a month, but have suddenly run into an issue I can't figure out.

Using Jellyfin on the Android TV app (Hisense), the 'grid direction' option for displaying media posters will default to 'horizontal' after being set to 'vertical'.

This issue only arose a few days ago, and I can't seem to figure it out. I can set all my library to 'vertical' but if I leave the app (for some period of time) and return, all libraries revert to 'horizontal'.  I tried changing other options such as Image Size and Image Type, and they also revert.

Other settings seem to stay as set.

thanks!

### Application version

0.15.5

### Where did you install the app from?

Google Play

### Device information

Hisense SmartTV 4K FFM

### Android version

10

### Jellyfin server version

10.8.8

**Comment by jezzom** [comment]
Just noticed that the Jellyfin app on my TV updated March 12th.  This seems to coincide with when this issue arose

**Comment by nielsvanvelzen** [comment]
Please provide app logs

**Comment by jezzom** [comment]
[log_20230315.log](https://github.com/jellyfin/jellyfin-androidtv/files/10988522/log_20230315.log)
[log_20230316.log](https://github.com/jellyfin/jellyfin-androidtv/files/10988523/log_20230316.log)
[log_20230314.log](https://github.com/jellyfin/jellyfin-androidtv/files/10988524/log_20230314.log)

These are logs from my server.  if the android TV app creates logs, i am not sure how to access them

**Comment by nielsvanvelzen** [comment]
You'll need to use logcat to get the logs from a device: https://wiki.lineageos.org/how-to/logcat. Make sure to reproduce the issue just before getting the logs.

**Comment by jezzom** [comment]
thanks.  I may or may not be smart enough to figure all this out before (hopefully) someone better than me can reproduce this problem. 
Of note, when I checked Jellyfin today (after being left overnight), the Vertical Grid Direction hadn't yet reverted to horizontal.  However, when i restarted my TV to troubleshoot, the Grid Direction did indeed reset again.
I'll see if I can somehow get proper logs

**Comment by jezzom** [comment]
ok just another update:  I went to open Jellyfin (android TV) after my PC restarted for a Windows update.  Somehow, without any input from me, the Grid Direction seems to have reset to how I had it before I started having the issue with 'Grid Direction reverting to horizontal'.  I restarted my TV (to troubleshoot) and indeed the Grid Direction now seems to (once again) properly save it's setting (as vertical).  

As the problem seems to have fixed itself somehow, I won't be able reproduce the issue and provide logs. 
thanks to nielsvanvelzen.  I will close this thread for now...
